frontend/public/components/utils/breadcrumbs.ts

```diff
@@ -13,13 +13,26 @@ export const getBreadcrumbPath = (match: any, customPlural?: string) => {
   return `/k8s/cluster/${customPlural || match.params.plural}`;
 };
 
-export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => [
-  {
-    name: `${kindObj.labelPlural}`,
-    path: getBreadcrumbPath(match),
-  },
-  {
-    name: i18next.t('details-page~{{kind}} details', { kind: kindObj.label }),
-    path: `${match.url}`,
-  },
-];
+export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => {
+  return match.url.startsWith('/k8s/cluster/config')
```

Comment:
@spadgett Is this too broad a change?  Can we say that any urls which start with '/k8s/cluster/config' should have a 'Global Config' breadcrumb?

Reply:
Arg..already found an exception:
`http://localhost:9000/k8s/ns/knative-serving/operator.knative.dev~v1alpha1~KnativeServing/knative-serving`
![image](https://user-images.githubusercontent.com/12733153/105059274-18c8fb00-5a45-11eb-9e96-a6ba821c058d.png)

Reply:
Instead of this, I would check if the resource is in the `config.openshift.io` API group and maybe that the name is `cluster`.

Reply:
Hmm..have to handle `withExtensions` & `GlobalConfigPageExtension`'s

Reply:
Updated to use `kindObj.apiGroup === 'config.openshift.io' && match.params.name === 'cluster'`, however still not working for `knative-serving` which is:
```
kindObj.apiGroup = "operator.knative.dev"
match.params.name = "knative-serving"
```
Another option would be creating a Details Page component for GlobalConfig resources, instead of using `DefaultDetailsPage`, would be same as `DefaultDetailsPage` but different breadcrumbs. We would have to update `app-contents.tsx`, not sure how we do that with dynamic `GlobalConfigPageExtension`'s?

Another option I was considering was adding a search param to the links to the Details pages from the Global Config page, something like "?backTo='globalConfig'" or "detailPageType='globalConfigDetail'" and we could dynamically set the `breadcrumbsFor` prop from [here](https://github.com/openshift/console/blob/f8ffd52597c86737547b2935063f71acd0ca59c7/frontend/public/components/default-resource.jsx#L147)

Reply:
Removed `&& match.params.name === 'cluster'` as Cluster Version's `match.params.name` equals `version`.

Reply:
> Removed `&& match.params.name === 'cluster'` as Cluster Version's `match.params.name` equals `version`.

I would leave the `cluster` check and manually pass the breadcrumbs as props in the other special cases (knative-serving and cluster version).

Reply:
@spadgett, 

- Most of the global config resources use DefaultDetailsPage which calls `breadcrumbsForDetailsPage()` which will set global config breadcrumbs via condition test `kindObj.apiGroup === 'config.openshift.io' && match.params.name === 'cluster'`.  
- Plugin resources will have url to details page: `...?backTo=globalConfig`, which will manually pass in breadcrumbs in `DefaultDetailsPage`.  
- ClusterVersion has a dedicated Details page which I pass in breadcrumbs.

Reply:
@dtaylor113 I'm unclear on the need for the query param. What are "plugin resources"? Do the special cases not have a dedicated details page?

Reply:
`knative-serving` is a `GlobalConfigPageExtension` what I'm calling a 'plugin resource'.  It doesn't have a dedicated detail page AFAIK.

Reply:
It looks like `KnativeServing` doesn't have a dedicated page. The right fix might be to add one, but that's really owned by the dev console team. I might fix the core breadcrumbs now and open a follow up low severity BZ for the `KnativeServing` breadcrumb.

An alternative is to have the breadcrumbs component check `GlobalConfigPageExtension`. It's starting to feel like too much of the config page logic is creeping into the breadcrumbs component, though. Even checking the API group there isn't ideal.

Reply:
Ok, I'll remove the code to handle the plugins, and I'll create a BZ for 'dev-console` to create a dedicated details page for `KnativeServing` and include the breadcrumbing code they should use.